test_image fails

## Description

I package Mathics for openSUSE. With release 6.0.1, when trying to run the tests during builds, we see the following issue with `test_image`:

```
[  168s] =================================== FAILURES ===================================
[  168s] ____________________ test_image[BinaryImageQ[hedy]-False-] _____________________
[  168s] 
[  168s] str_expr = 'BinaryImageQ[hedy]', str_expected = 'False', msg = '', message = ''
[  168s] 
[  168s]     @pytest.mark.skipif(
[  168s]         os.getenv("SANDBOX", False),
[  168s]         reason="Test doesn't work in a sandboxed environment with access to local files",
[  168s]     )
[  168s]     @pytest.mark.parametrize(("str_expr, str_expected, msg"), image_tests)
[  168s]     def test_image(str_expr: str, str_expected: str, msg: str, message=""):
[  168s]         result = evaluate(str_expr)
[  168s]         if result is not SymbolNull or str_expected is not None:
[  168s]             expected = evaluate(str_expected)
[  168s]             if msg:
[  168s]                 assert result == expected, msg
[  168s]             else:
[  168s] >               assert result == expected
[  168s] E               assert <Expression: System`BinaryImageQ[-Image-]> == <SymbolConstant: System`False>
[  168s] 
[  168s] test/builtin/drawing/test_image.py:69: AssertionError
[  168s] ________________ test_image[BinaryImageQ[Binarize[hedy]]-True-] ________________
[  168s] 
[  168s] str_expr = 'BinaryImageQ[Binarize[hedy]]', str_expected = 'True', msg = ''
[  168s] message = ''
[  168s] 
[  168s]     @pytest.mark.skipif(
[  168s]         os.getenv("SANDBOX", False),
[  168s]         reason="Test doesn't work in a sandboxed environment with access to local files",
[  168s]     )
[  168s]     @pytest.mark.parametrize(("str_expr, str_expected, msg"), image_tests)
[  168s]     def test_image(str_expr: str, str_expected: str, msg: str, message=""):
[  168s]         result = evaluate(str_expr)
[  168s]         if result is not SymbolNull or str_expected is not None:
[  168s]             expected = evaluate(str_expected)
[  168s]             if msg:
[  168s]                 assert result == expected, msg
[  168s]             else:
[  168s] >               assert result == expected
[  168s] E               assert <Expression: System`BinaryImageQ[-Image-]> == <SymbolConstant: System`True>
[  168s] 
[  168s] test/builtin/drawing/test_image.py:69: AssertionError
...
[  168s] =========================== short test summary info ============================
[  168s] FAILED test/builtin/drawing/test_image.py::test_image[BinaryImageQ[hedy]-False-]
[  168s] FAILED test/builtin/drawing/test_image.py::test_image[BinaryImageQ[Binarize[hedy]]-True-]
```

## How to Reproduce

Run tests with `pytest` (Python 3.9/3.10).

## Expected behavior

Tests completing successfully.

## Your Environment

Build against the following:
* Python 3.9/3.10 (error seen with both)
* NumPy: 1.24.2 
* SymPy: 1.11.1
* SciPy: 1.10.1
* Mathics-Scanner 1.13.0

In addition

**Desktop (please complete the following information):**
 - openSUSE Tumbleweed (Linux x86_64)
 - Browser: NA

Thanks in advance for any suggestions or work-arounds. Thanks also for the great software.

--- a/test/builtin/drawing/test_image.py
+++ b/test/builtin/drawing/test_image.py
@@ -9,6 +9,7 @@ from test.helper import evaluate
 
 import pytest
 
+from mathics.builtin.base import check_requires_list
 from mathics.core.symbols import SymbolNull
 
 # Note we test with tif, jpg, and gif. Add others?
@@ -50,10 +51,10 @@ image_tests = [
 ]
 
 
-# @pytest.mark.skipif(
-#     not check_requires_list(["skimage"]),
-#     reason="scikit-image (AKA skimage) is needed for working with Images",
-# )
+@pytest.mark.skipif(
+    not check_requires_list(["skimage"]),
+    reason="Test doesn't work in a when scikit-image is not installed",
+)
 @pytest.mark.skipif(
     os.getenv("SANDBOX", False),
     reason="Test doesn't work in a sandboxed environment with access to local files",

--- a/test/builtin/numbers/test_calculus.py
+++ b/test/builtin/numbers/test_calculus.py
@@ -8,7 +8,7 @@ FindRoot[], FindMinimum[], NFindMaximum[] tests
 
 
 """
-from test.helper import check_evaluation, evaluate
+from test.helper import check_evaluation
 from typing import Optional
 
 import pytest
@@ -127,7 +127,8 @@ def test_integrate(str_expr: str, str_expected: str, assert_fail_message):
             "D[{y, -x}[2], {x, y}]",
             "D[{y, -x}[2], {x, y}]",
             [
-                "Multiple derivative specifier {x, y} does not have the form {variable, n}, where n is a non-negative machine integer."
+                "Multiple derivative specifier {x, y} does not have the form {variable,"
+                " n}, where n is a non-negative machine integer."
             ],
         ),
     ],  # Issue #502

